R/correctionsataglance-render.R

```diff
@@ -116,14 +116,28 @@ plotLanes <- function(gsplotObject, laneData, whichCol,
              labels = laneData[[labelName]], cex=1) 
     }
     
-    if(any(names(laneData) %in% 'numText')){  
+    if(any(names(laneData) %in% 'numText')){   
       i <- which(!is.na(laneData$numText))
+      if (any(laneData$moveText)) { 
+        pos <- NA
+        dateRange <- format(addData$dateData$dateRange, "%m/%d/%Y")
+        xyText <- format(laneData$xyText$x, "%m/%d/%Y")
+          if (any(xyText == dateRange[1])) {
```

Comment:
shouldn't we be checking if it is <= the start date? And similarly, if it is >= the end date on line 128?